js/adapt-contrib-flipcard.js

```diff
@@ -1,191 +1,238 @@
-define([
-  'coreViews/componentView',
-  'coreJS/adapt',
-  'underscore'
-], function(ComponentView, Adapt){
-
-  class Flipcard extends ComponentView {
-
-    events() {
-      return {
-        'click .flipcard__item-face': 'onClickFlipItem',
-        'keypress .flipcard__item': 'onClickFlipItem'
-      }
+import ComponentView from 'coreViews/componentView';
+import Adapt from 'coreJS/adapt';
+import _ from 'underscore';
+
+class Flipcard extends ComponentView {
+
+  events() {
+    return {
+      'click .flipcard__item': 'onClickFlipItem',
+      'keypress .flipcard__item': 'onClickFlipItem'
+    };
+  }
+
+  preRender() {
+    this.model.set('_items',
+      this.model.get('_items').map(obj => ({
+        ...obj,
+        backBodyText: $(obj.backBody).text()
+      }))
+    );
+    // is apple platform used to determine if voiceover is being used
+    const platform = navigator?.platform?.toLowerCase();
+    this.isApplePlatform = (platform === 'mac' || platform === 'macintel' || platform === 'iphone' || platform === 'ipad');
```

Comment:
I think you can use
`if (Adapt.device.getOperatingSystem() === 'mac' || Adapt.device.isAppleDevice())`
It uses the same logic as what you are writing, but if it gets improved in the future we will be falling back to it